config/locales/pages/de/about.markdown

```diff
@@ -0,0 +1,60 @@
+# About CrimethInc.
+
+## What Is Crimethink?
+
+**Crimethink is everything that evades control:** the daydream in the classroom, the renegade breaking ranks, the spray-painted walls that continue to speak even under martial law. It is the persistent sense that things could be otherwise, that there is nothing natural or inevitable about the prevailing social order. In a world optimized for administration, everything that cannot be classified or displayed on a screen is crimethink. It is the spirit of rebellion without which freedom is literally [unthinkable](https://moxie.org/blog/we-should-all-have-something-to-hide).
+
+## What Is CrimethInc.?
+
+**CrimethInc. is a rebel alliance** — a secret society pledged to the propagation of crimethink. It is a think tank producing inflammatory ideas and action, a sphinx posing questions fatal to the superstitions of our age.
+
+**CrimethInc. is a banner for anonymous collective action.** It is not a membership organization, but a mouthpiece for longings that extend throughout the population at large. Anyone can be CrimethInc. — it could be your next-door neighbor or the person sitting beside you on the bus. You and your friends already constitute an [affinity group](http://destructables.org/node/54), the organizational model best suited to guerrilla tactics, ready to go into action against all the forces that threaten your freedom.
+
+**CrimethInc. is an international network of aspiring revolutionaries** extending from Kansas to Kuala Lumpur. For over twenty years, we have published news, analysis, [books](/books), [journals](/journals), [posters](/posters), [videos](/videos), [podcasts](/podcast), and a wide range of other resources — all copyright free, produced and distributed by volunteer labor, without reliance on external funding or market trends. We also coordinate [speaking tours](/2015/12/28/report-to-change-everything-us-tour), [debates](/2012/09/17/post-debate-debrief-video-and-libretto), and various other public events. Though we rarely seek public recognition for our efforts, everything we do is informed by our participation in social movements.
+
+**CrimethInc. is a desperate venture.** As this society lurches ever closer to annihilation, we are staking everything on the possibility that we could kick open the escape hatch to another future. Rather than competing for social capital or selling ourselves to the highest bidder, we have thrown ourselves completely into the struggle for a better world. We invite you to do the same.
+
+## General Inquiries
+
+_Email:_ \\
+[hello@crimethinc.com](mailto:hello@crimethinc.com)
+
+_Twitter:_ \\
+[@crimethinc](https://twitter.com/crimethinc)
+
```

Comment:
@veganstraightedge should we put the fb page here since it is active and respond pretty fast.